### ISSUE TITLE
Race condition but only in chromatic?

### ISSUE THREAD

**Issue by fazulk** [mswjs/msw-storybook-addon#80]
So far no issues running storybook locally via `start-storybook`, 

but seeing a race condition when building storybook for chromatic. Looks like The interceptor gets loaded after the story tried to access apis.

Here is my current setup
```
    "@storybook/vue3": "^6.4.20",
    "@storybook/builder-vite": "0.1.22",
    "msw": "^0.39.2",
    "msw-storybook-addon": "^1.6.1",
    "chromatic": "^6.5.3"
```

And in my `preview.js`

```import { initialize, mswDecorator } from 'msw-storybook-addon'
import { handlers } from '../mocks/handlers'

initialize()

export const decorators = [mswDecorator]

export const parameters = {  
  msw: {
    handlers,
  },
}
```

`handlers` is an array of `RestHandler` endpoints

When loading the built story:
![image](https://user-images.githubusercontent.com/21964900/162028553-60a898eb-765c-4899-8975-ccd882d8b39d.png)

Just to see i also tried loading msw into the parameters of the individual story, with no luck.

**Comment by fazulk** [comment]
Building stories, and serving locally works, it's just an issue with chromatic from what I can tell. Perhaps i should post this issue over there, if the above code looks fine?

**Comment by fazulk** [comment]
I resolved this by upgrading vite from `2.8.6` to `2.9.1`.
